src/entity/users.rs

```diff
@@ -1,5 +1,7 @@
-use sea_orm::entity::prelude::*;
-use serde::Deserialize;
+use sea_orm::{entity::prelude::*, Set};
+use serde::{Deserialize};
```

Comment:
неправильное оформление кода
```suggestion
use serde::Deserialize;
```